exercises/Fibonacci/Python/main.py

```diff
@@ -0,0 +1,14 @@
+n = int(input("Enter an integer :"))

```

Comment:
This file is currently something of a monolith, try breaking it up into functions. For example, one function to get user input, one function to compute the nth fibonacci number, and another one to display the output, and then a `main` function to run the actual logic of calling each of these functions. This will make it easier for you to re-use these code bits in the future, and also will make the code easier to navigate and document (each function can have a code comment description)